docs/cloud/azureai/use-flow-in-azure-ml-pipeline.md

```diff
@@ -227,19 +321,39 @@ Given above, if your flow has logic relying on identity or environment variable,
-|                  | retry_settings.timeout                              | the same as above                                | integer                 | Define the timeout in seconds for executing custom run() function. If the execution time is higher than this threshold, the mini-batch will be aborted, and marked as a failed mini-batch to trigger retry. |
-|                  | logging_level                                       | the same as above                                | INFO, WARNING, or DEBUG | Define which level of logs will be dumped to user log files. |
+### Input ports
+
+| key         | source | type                   | description                                                  |
+| ----------- | ------ | ---------------------- | ------------------------------------------------------------ |
+| data        | fixed  | uri_folder or uri_file | required; to pass in input data. Supported format includes [`mltable`](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-mltable?view=azureml-api-2&tabs=cli#authoring-mltable-files) and list of jsonl files. |
+| run_outputs | fixed  | uri_folder             | optional; to pass in output of a standard flow for [an evaluation flow](../../how-to-guides/develop-a-flow/develop-evaluation-flow.md). Should be linked to a `flow_outputs` of a previous flow node in the pipeline. |
+
+### Output ports
+
+| key          | source | type       | description                                                  |
+| ------------ | ------ | ---------- | ------------------------------------------------------------ |
+| flow_outputs | fixed  | uri_folder | an uri_folder with 1 or more jsonl files containing outputs of the flow runs |
+| debug_info   | fixed  | uri_folder | an uri_folder containing debug information of the flow run, e.g., run logs |
+
+### Parameters
+
+| key                                                 | source                                           | type   | description                                                  |
+| --------------------------------------------------- | ------------------------------------------------ | ------ | ------------------------------------------------------------ |
+| \<flow-input-name\>                                 | from flow inputs                                 | string | default value will be inherited from flow inputs; used to override [column mapping](../../how-to-guides/run-and-evaluate-a-flow/use-column-mapping.md) for flow inputs. |
+| connections.\<node-name\>.connection                | from nodes of built-in LLM tools                 | string | default value will be current value defined in `flow.dag.yaml` or `run.yaml`; to override used connections of corresponding nodes. Connection should exist in current workspace. |
```

Comment:
why we say this is from built-in LLM tools?

Reply:
for explicit `Connection` inputs, their keys are not fixed